R/pb_download.R

```diff
@@ -11,7 +11,7 @@
 #'
 #' @export
 #' @examples \donttest{
-#'  try({ # this try block is to avoid errors on CRAN, not needed for normal use
+#' \dontshow{try(\{}
```

Comment:
I read through the examplesIf [docs](https://roxygen2.r-lib.org/articles/rd.html#examples) and stole this from there. Could still do if NOT_CRAN but I think this is fine